code/AssetLib/Assbin/AssbinLoader.cpp

```diff
@@ -91,7 +91,12 @@ bool AssbinImporter::CanRead(const std::string &pFile, IOSystem *pIOHandler, boo
     }
 
     char s[32];
-    in->Read(s, sizeof(char), 32);
+    const size_t read = in->Read(s, sizeof(char), 32);
+
+    pIOHandler->Close(in);
```

Comment:
Why do we close the handler here and below?

Reply:
Good catch. I think we should close here before returning.

Reply:
Good catch. I will merge.